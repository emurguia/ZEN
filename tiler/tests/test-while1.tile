// This tests that the while loop is working as expected

init {
	int x;
	int y;

	tile(3,3);

	x = 7;
	while (x > 0){
		iprint(x);
		x = x -1;
	}

	y = 100;
	while(y < 1){
		sprint("While loop should not have been entered?!?!");
	}

	close();
}