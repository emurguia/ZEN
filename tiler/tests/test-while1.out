7
6
5
4
3
2
1
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.